src/main/resources/uischema/update.json

```diff
@@ -0,0 +1,286 @@
+                      {
+                        "type": "attribute",
+                        "value": {
+                          "name": "columnValue",
+                          "displayName": "Column Value",
+                          "inputType": "stringOrExpression",
+                          "required": false,
+                          "helpTip": ""
+                        }
+                      },
+                      {
+                        "type": "attribute",
+                        "value": {
+                          "name": "propertyType",
+                          "displayName": "Column Type",
+                          "inputType": "stringOrExpression",
+                          "defaultValue": "VARCHAR",
+                          "required": false,
+                          "helpTip": ""
+                        }
+                      }
+                    ],
+                    "inputType": "ParamManager",
+                    "onValueChange": {
+                      "function": "buildQuery",
+                      "params": [
+                        {
+                          "queryType": "update",
+                          "resultField": "query"
+                        }
+                      ]
+                    },
+                    "parentField": "table"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "type": "attributeGroup",
+            "value": {
+              "groupName": "Advanced",
+              "elements": [
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "queryTimeout",
+                    "displayName": "Query Timeout",
+                    "inputType": "stringOrExpression",
+                    "defaultValue": "",
+                    "required": "false",
+                    "helpTip": "Query timeout in seconds"
+                  }
+                },
+                {
+                  "type": "attribute",
+                  "value": {
+                    "name": "transactionIsolation",
+                    "displayName": "Transaction Isolation Level (Executes in a separate transaction regardless of sequence transaction)",
+                    "inputType": "combo",
+                    "comboValues": [
+                      "TRANSACTION_NONE",
+                      "TRANSACTION_READ_COMMITTED",
+                      "TRANSACTION_READ_UNCOMMITTED",
+                      "TRANSACTION_REPEATABLE_READ",
+                      "TRANSACTION_SERIALIZABLE"
+                    ],
+                    "defaultValue": "TRANSACTION_NONE",
+                    "required": "false",
+                    "helpTip": "Transaction isolation level"
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "type": "attributeGroup",
+            "value": {
+              "groupName": "Operation Result",
```

Comment:
Shall we call this as "Output"? Please check the HTTP connector and align the terminology with that.

Reply:
Please check other places as well.